### ISSUE TITLE
[JENKINS-33822] New Item creation keyboard navigation does not work

### ISSUE THREAD

**Issue by jenkins-infra-bot** [timja-org/jenkins-gh-issues-poc-11-23#5607]
<p>We'll need to make sure keyboard navigation works on this. Currently the radio buttons are display: none, which makes the browser not allow selection.</p>

---
<details><summary><i>Originally reported by <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/kzantow.png" title="kzantow's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=kzantow">kzantow</a>, imported from: <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-33822" target="_blank">New Item creation keyboard navigation does not work</a></i></summary>
<i><ul>
<li><b>assignee</b>: <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/recena.png" title="recena's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=recena">recena</a>
<li><b>status</b>: Resolved
<li><b>priority</b>: Major
<li><b>component(s)</b>: core
<li><b>label(s)</b>: 2.0, 2.0-beta, community-bee, regression
<li><b>resolution</b>: Fixed
<li><b>resolved</b>: 2016-05-03T13:37:55+00:00
<li><b>votes</b>: 0
<li><b>watchers</b>: 5
<li><b>imported</b>: 2025-11-23
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>We'll need to make sure keyboard navigation works on this. Currently the radio buttons are display: none, which makes the browser not allow selection.</p></pre>
</details>
</details>
<ul><li><i>environment</i>: <code>2.0-beta</code></li></ul>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/danielbeck.png" title="danielbeck's avatar" /> danielbeck:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-33822?focusedId=252033&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-252033">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Please indicate which version this is about. I'm pretty sure it's not one that has actually been released. There are tons of radio buttons in 1.654.</p>
</pre>
</details>

</ul>
</details>

<p>Please indicate which version this is about. I'm pretty sure it's not one that has actually been released. There are tons of radio buttons in 1.654.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>gusreiber:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-33822?focusedId=253763&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-253763">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=kzantow" class="user-hover" rel="kzantow">kzantow</a> happy to take it. Pretty straight forward, I think.</p>
</pre>
</details>

</ul>
</details>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=kzantow" class="user-hover" rel="kzantow">kzantow</a> happy to take it. Pretty straight forward, I think.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/scm_issue_link.png" title="scm_issue_link's avatar" /> scm_issue_link:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-33822?focusedId=256233&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-256233">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Code changed in jenkins
User: Manuel Recena
Path:
 core/src/main/resources/hudson/model/View/newJob.jelly
 war/src/main/js/add-item.js
 war/src/main/js/widgets/add/addform.less
<a href="http://jenkins-ci.org/commit/jenkins/ece6606066abcf33dbff7be42b35e78253cad267" class="external-link" target="_blank" rel="nofollow noopener">http://jenkins-ci.org/commit/jenkins/ece6606066abcf33dbff7be42b35e78253cad267</a>
Log:
  <a href="https://issues.jenkins.io/browse/JENKINS-33822" title="New Item creation keyboard navigation does not work" class="issue-link" data-issue-key="JENKINS-33822"><del>JENKINS-33822</del></a> Create a new item using the keyboard</p>


</pre>
</details>

</ul>
</details>

<p>Code changed in jenkins<br/>
User: Manuel Recena<br/>
Path:<br/>
 core/src/main/resources/hudson/model/View/newJob.jelly<br/>
 war/src/main/js/add-item.js<br/>
 war/src/main/js/widgets/add/addform.less<br/>
<a href="http://jenkins-ci.org/commit/jenkins/ece6606066abcf33dbff7be42b35e78253cad267" class="external-link" target="_blank" rel="nofollow noopener">http://jenkins-ci.org/commit/jenkins/ece6606066abcf33dbff7be42b35e78253cad267</a><br/>
Log:<br/>
  <a href="https://issues.jenkins.io/browse/JENKINS-33822" title="New Item creation keyboard navigation does not work" class="issue-link" data-issue-key="JENKINS-33822"><del>JENKINS-33822</del></a> Create a new item using the keyboard</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/scm_issue_link.png" title="scm_issue_link's avatar" /> scm_issue_link:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-33822?focusedId=256235&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-256235">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Code changed in jenkins
User: Daniel Beck
Path:
 core/src/main/resources/hudson/model/View/newJob.jelly
 war/src/main/js/add-item.js
 war/src/main/js/widgets/add/addform.less
<a href="http://jenkins-ci.org/commit/jenkins/eae8cd6fb3cc4ff1bb92b037d6830686d56a31e5" class="external-link" target="_blank" rel="nofollow noopener">http://jenkins-ci.org/commit/jenkins/eae8cd6fb3cc4ff1bb92b037d6830686d56a31e5</a>
Log:
  Merge pull request #2250 from recena/<a href="https://issues.jenkins.io/browse/JENKINS-33822" title="New Item creation keyboard navigation does not work" class="issue-link" data-issue-key="JENKINS-33822"><del>JENKINS-33822</del></a></p>

<p><a href="https://issues.jenkins.io/browse/JENKINS-33822" title="New Item creation keyboard navigation does not work" class="issue-link" data-issue-key="JENKINS-33822"><del>JENKINS-33822</del></a> Create a new item using the keyboard</p>


<p>Compare: <a href="https://github.com/jenkinsci/jenkins/compare/d442682699b6...eae8cd6fb3cc" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/jenkins/compare/d442682699b6...eae8cd6fb3cc</a></p>

</pre>
</details>

</ul>
</details>

<p>Code changed in jenkins<br/>
User: Daniel Beck<br/>
Path:<br/>
 core/src/main/resources/hudson/model/View/newJob.jelly<br/>
 war/src/main/js/add-item.js<br/>
 war/src/main/js/widgets/add/addform.less<br/>
<a href="http://jenkins-ci.org/commit/jenkins/eae8cd6fb3cc4ff1bb92b037d6830686d56a31e5" class="external-link" target="_blank" rel="nofollow noopener">http://jenkins-ci.org/commit/jenkins/eae8cd6fb3cc4ff1bb92b037d6830686d56a31e5</a><br/>
Log:<br/>
  Merge pull request #2250 from recena/<a href="https://issues.jenkins.io/browse/JENKINS-33822" title="New Item creation keyboard navigation does not work" class="issue-link" data-issue-key="JENKINS-33822"><del>JENKINS-33822</del></a></p>

<p><a href="https://issues.jenkins.io/browse/JENKINS-33822" title="New Item creation keyboard navigation does not work" class="issue-link" data-issue-key="JENKINS-33822"><del>JENKINS-33822</del></a> Create a new item using the keyboard</p>


<p>Compare: <a href="https://github.com/jenkinsci/jenkins/compare/d442682699b6...eae8cd6fb3cc" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/jenkins/compare/d442682699b6...eae8cd6fb3cc</a></p>
